eagleproject/core/blockchain/harvest/snapshots.py

```diff
@@ -88,9 +94,70 @@ def isbetween(start, end, v):
         return False
     return True
 
+def _build_asset_block_oeth(symbol, block_number):
+    strat_holdings = {}
+
+    # TODO: Rename these columns??
```

Comment:
I think it's safer to keep it on this PR, but definitely something we need to investigate and look into in the future

Reply:
Gotcha